pkg/sensors/tracing/matchbinaries_follow_children_test.go

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package tracing
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"testing"
+	"time"
+
+	"github.com/cilium/tetragon/pkg/grpc/tracing"
+	"github.com/cilium/tetragon/pkg/k8s/apis/cilium.io/v1alpha1"
+	"github.com/cilium/tetragon/pkg/logger"
+	"github.com/cilium/tetragon/pkg/reader/notify"
+	"github.com/cilium/tetragon/pkg/testutils"
+	"github.com/cilium/tetragon/pkg/testutils/perfring"
+	tus "github.com/cilium/tetragon/pkg/testutils/sensors"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMatchBinariesFollowChildren(t *testing.T) {
+
+	testutils.CaptureLog(t, logger.GetLogger().(*logrus.Logger))
+	ctx, cancel := context.WithTimeout(context.Background(), tus.Conf().CmdWaitTime)
+	defer cancel()
+	shPath, err := exec.LookPath("sh")
+	if err != nil {
+		t.Fatalf("failed to find 'sh' exec: %v", err)
+	}
+	tmpShPath, err := testutils.CopyFileToTmp(shPath)
+	if err != nil {
+		t.Fatalf("failed to copy 'sh' exec: %v", err)
+	}
+	t.Cleanup(func() {
+		// os.Remove(tmpShPath)
```

Comment:
I think you want to uncomment that, or pass `t` in your `testutils.CopyFileToTmp` to register the cleanup callback so that the caller doesn't need to.

Reply:
Yap, need to uncomment that. I guess it was a leftover.